lib/utils/status.js

```diff
@@ -1,10 +1,20 @@
 'use strict';
 
+const logger = require('webpack-log');
 const colors = require('./colors');
 const runOpen = require('./runOpen');
 
 // TODO: don't emit logs when webpack-dev-server is used via Node.js API
 function status(uri, options, log, useColor) {
+  if (options.quiet === true) {
+    // Add temporary logger to output just the status of the dev server
+    log = logger({
```

Comment:
Why don't use `log`?

Reply:
Assuming you want `logger` to be `log`, because `log` is also the function argument for status function.

Reply:
Can you clarify?

Reply:
When `--quiet` is passed, The `log` received by the status function would have log level as `silent`, due to which `log.info` would not print anything to `stdout`. That's why overriding the passed log so that only status is printed + original log remain unaffected for rest of the modules. 

Hope that explains the solution.

Reply:
hm, look we can fix it in master, only for next

Reply:
I am afraid but I didn't get what you meant above?

Reply:
@rishabh3112 all fine, latest webpack has logger and use other logic we need switch on this logic and rewrite some places

Reply:
I can help if you can point me where changes has to be done

Reply:
@rishabh3112 we will refactor this in next version, can't be node in master right now

Reply:
node?

Reply:
@rishabh3112 no, new version of webpack has logger, we need migrate on this logger instead using `webpack-log`

Reply:
Got it. Thanks for clarifying.
Is master fine with webpack-log or should I close this PR ?

Reply:
@rishabh3112 All good here